docs/faq.md

```diff
@@ -74,6 +74,8 @@ If you're already publishing your module with TypeScript, you might be using NPM
 
 If you're using some other set up, just run `typings install` before you execute the build step. This will install the type definitions from `typings.json` before the TypeScript compiler runs.
 
+Therefore, in normal circumstances you don't need to commit the typings folder into your version control. Unless you have problem accessing the typings (e.g. your build server is blocked from accessing the Internet).
```

Comment:
You could probably revert this wording to the previous wording and just move it to the top. I think it's more direct than the instructions I wrote before.